Create-Login-and-Register-System-with-C#-Winforms/index.md

```diff
@@ -0,0 +1,430 @@
+            ConnectionType = Table;
+            dt = new DataTable(ConnectionType);
+            try
+            {
+                string command = RecordSource.ToUpper();
+
+                //======================if sql contains select==========================================
+                MySqlDataAdapter da2 = new MySqlDataAdapter(RecordSource, connection);
+
+                DataSet tempds = new DataSet();
+                da2.Fill(tempds, ConnectionType);
+                da2.Fill(tempds);
+
+                //======================================================================================
+
+
+            }
+            catch (Exception err) { MessageBox.Show(err.Message); }
+        }
+
+        // function to bring selected results based on column name and row index
+        public string Results(int ROW, string COLUMN_NAME)
+        {
+            try
+            {
+                return dt.Rows[ROW][COLUMN_NAME].ToString();
+            }
+            catch (Exception err)
+            {
+                MessageBox.Show(err.Message);
+                return "";
+
+            }
+        }
+
+        // function to bring selected results based on column index and row index
+        public string Results(int ROW, int COLUMN_NAME)
+        {
+            try
+            {
+                return dt.Rows[ROW][COLUMN_NAME].ToString();
+            }
+            catch (Exception err)
+            {
+                MessageBox.Show(err.Message);
+                return dt.Rows[ROW][COLUMN_NAME].ToString();
+
+            }
+        }
+
+        // Execute select statement
+        public void ExecuteSelect(string Sql_command)
+        {
+            RecordSource = Sql_command;
+            ConnectionType = Table;
+
+            dt = new DataTable(ConnectionType);
+            try
+            {
+                string command = RecordSource.ToUpper();
+                MySqlDataAdapter da = new MySqlDataAdapter(RecordSource, connection);
+                ds = new DataSet();
+                da.Fill(ds, ConnectionType);
+                da.Fill(dt);
+                tempdata = new DataGridView();
+            }
+            catch (Exception err)
+            {
+                MessageBox.Show(err.Message);
+            }
+
+
+        }
+
+        // count Number of rows after selecy
+        public int Count()
+        {
+            return dt.Rows.Count;
+        }
+    }
+}
+
+```
+
+- Now that we have Config.cs we can execute any MySQL statement 
+
+### Step IV: Create Register Windows form.
+
+From Microsoft Visual Studio create a windows form by clicking on PROJECT -> Add Windows Form, give the form a name `Register`  then click Add.
+
+We have two Windows form classes that is Form1.cs and Register.cs.
+
+### Step V: Design Login and Register interface.
+
+### Login form
+
+- Click on Form1.cs in Solution Explorer and on the form that displays add 3 buttons, 2 textboxes, and 2 labels.
```

Comment:
This would be better if you split it into two different steps.

Reply:
it is a design that can be done within one step.
Thanks